### PR TITLE
Add section on CDN caching and authentication.

### PR DESCRIPTION
**Pull request by cmc333333**
After being bit by an incorrect mental model of the CDN's caching behavior, I
figured we should document it. At roughly the same time, someone asked about
authentication, so I added that as well.

Maintainers should be able to edit this PR directly -- feel free to rewrite everything :)

### CODE REVIEW

**Comment by brittag** [comment]
Thank you so much for this addition - yes, I think this will be really helpful and prevent other people from running into problems!

The one simple change I'd make is moving these new "Caching" and "Authentication" sections together underneath something like a "More about how this service works" section, and then sticking that section after the "External services and applications" section - so that you get the three sets of basic creation/update instructions first, and then additional info about more advanced aspects. I'll wait for a double check on the technical info by Atlas before mucking around with content though. :)

File: content/docs/services/cdn-route.md [hunk 1]
```diff
@@ -123,6 +123,27 @@ field. If you do, follow ["How to set up DNS"](#how-to-set-up-dns) again.
...
+CloudFront [uses](http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Expiration.html)
+your application's `Cache-Control` or `Expires` HTTP headers to determine how
+long to cache content. If your application does not provide these headers,
+CloudFront will use a default timeout of **24 hours**. This can be
+particularly confusing as different requests might be routed to different
+CloudFront Edge endpoints.
+
+While there is no mechanism for cloud.gov users to trigger a cache clear, the
+cloud.gov support team can. Note however, that cache invalidation is not
+instantaneous; Amazon expecting a lag time of 10-15 minutes (more if there are
```
Comment (fureigh) [inline content/docs/services/cdn-route.md]: @cmc333333 Should `Amazon expecting` simply be `expect`?

Comment (cmc333333) [reply content/docs/services/cdn-route.md]: Sounds good. Or "Amazon recommends expecting".

**Comment by cmc333333** [comment]
Hey @brittag and @fureigh, would you mind making these changes? I'm going to be bogged down in other work for a bit.

**Comment by brittag** [comment]
Thanks @cmc333333, we'll take this forward!

**Comment by mogul** [comment]
@jmcarp can you take a look at this one?

**Review by jmcarp — DISMISSED** [review]

**Comment by jmcarp** [comment]
Technical content looks good to me.

**Comment by brittag** [comment]
@jmcarp Now ready from a content perspective! Feel free to approve again and merge.

**Review by jmcarp — APPROVED** [review]

**Pull request merged**